components/monitoring/prometheus/README.md

```diff
@@ -1,34 +1,67 @@
-## Data Plane Clusters Prometheus Instances
-We use the Openshift-provided Prometheus deployments, platform and
-user-workload-monitoring (UWM), alongside a Prometheus instance deployed by the RHOBS
+## RHTAP Central Monitoring
+The RHTAP monitoring solution is based on three Prometheus instances deployed to each
+Production and Staging cluster. Each cluster writes a subset of the metrics it generates
```

Comment:
does this include the host cluser?

Reply:
It does.

Reply:
perhaps make the text explicit about that

Reply:
done